index.js

```diff
@@ -1,18 +1,49 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
+console.log(header)
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerDiv = document.querySelector(".traveler")
+
+const travelerImg = travelerDiv.querySelector("img")
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
 
+const travelerName = travelerDiv.querySelector("h2")
+travelerName.textContent = traveler.name
 
+const travelerNickame = travelerDiv.querySelector("em")
+travelerNickame.textContent = traveler.nickname
```

Comment:
Nice work scoping the search to the `travelerDiv`. Scoping our searches is also better performance wise because there's less DOM traversal needed to be done!